pkg/workspace/operator.go

```diff
@@ -0,0 +1,30 @@
+package workspace
```

Comment:
Personally I would suggest you only include API types definition in this PR, operator code can be a separate PR.

Reply:
That's cool and I'll change it.